### DISCUSSION TITLE
Crazyflie not taking off

### DISCUSSION THREAD

**Discussion by SharyatSB** [bitcraze/discussions#920] [Q&A]
Hey all, 

I am trying to control my crazyflie however it does not seem to be taking off. I have tried to control using my ios app and even at 100% thrust it does not take off. I also tried to use the take off button on the client and even then it did not take off. I have rechecked propeller placement and everything seems to be good. I am attaching a screenshot of multiple propeller tests that I conducted on the client and seems like all motors failed. 

Edit1:  I should also mention that theclient shows the percent of thurst and motors etc while flying. The motors are usually at 70% but the thrust is always 0%. Not sure if this will help. 

![CFerror1](https://github.com/bitcraze/discussions/assets/77988600/e2eea93f-89f5-45b5-bcd9-94ff35aec321)

Thanks for the help!

**Comment by tobbeanton** [comment]
Have you checked that the propellers are installed correctly? They are probably up-side-down.
